src/_articles/language/mixins.md

```diff
@@ -1,29 +1,31 @@
 ---
-title: "Mixins in Dart"
-description: "Mixins let you implement functionality once and use it in multiple classes."
+title: A Brief History of Mixins in Dart
+description: Mixins let you implement functionality once and use it in multiple classes.
 written: 2012-12-18
-updated: 2016-08-25
+updated: 2018-06-29
 category: language
-obsolete: true
 ---
 
 _Written by Gilad Bracha <br>
-December 2012 (updated August 2016)_
+December 2012 (updated June 2018)_
 
-This document describes mixins in Dart. We have relaxed
-some of the restrictions of early implementations; this document
-describes the current state of play.
+Looking for an introduction to mixins?
+See the Language Tour section on [Adding features to a class: mixins][].
 
-Dart 1.13 and greater supports mixins that can extend from classes
+This article provides a brief theoretical presentation of the evolution of mixins in Dart.
+
+Feature evolution notes:
+
+- Support for `--supermixin` is being rescinded in Dart 2.
```

Comment:
Let's avoid "rescinded", since it's not a commonly used word. How about:

Dart 2 does not support `--supermixin`.